source/css/_partial/sidebar.styl

```diff
@@ -40,8 +40,8 @@
       &:focus
         color color-font
 
-  .tag
-    small
+  .archive-list-count,
+  .tag small
```

Comment:
In fact, browsers generally provide a smaller font size for the `<small>` element, so there might be some inconsistency here. However, I will merge this pull request. Thank you for your patience!